ddl/column.go

```diff
@@ -135,6 +135,11 @@ func (d *ddl) onAddColumn(t *meta.Meta, job *model.Job) (ver int64, _ error) {
 		return ver, errors.Trace(err)
 	}
 
+	if err = checkAddColumnTooManyColumns(len(tblInfo.Columns)); err != nil {
```

Comment:
Just wondered， is it ok if we can move this snippet up to line 129? Per my naive understanding, `tblInfo` is ready to be used at line 129, hence validating check can be done at then.